ci/browser-tests/cypress/integration/contract_mm.ci.spec.js

```diff
@@ -0,0 +1,38 @@
+context("Contracts MM", () => {
+    beforeEach(() => {
+        cy.dummyLogin("benjamin");
+        cy.selectLanguage("#EN");
+    });
+
+    it("finds contracts with search and navigates there + edits details + project link", () => {
+        cy.visit("#/contracts");
+        cy.get("[data-cy='search-shortcut-all-contracts']").click();
+        // cy.get("[data-cy='toggle-filters-visibility']").click(); // uncomment if default is hidden, has been in flux
+        cy.get("input[id='contract-filter-input/:contract-name']").type("TEPPO");
+
+        // cy.get("[data-cy='expand-contracts']").click({force: true});
+        // cy.get("[data-cy='expand-contracts'] span").first().click();
```

Comment:
Commented code, if it doesn't have any purpose (explaining something, for instance) then to be removed.